client/templates/Admin/Stats/metrics.html.twig

```diff
@@ -9,140 +9,135 @@
-        {{ form_start(form) }}
-
-        {% set customDateRange %}
-            <div class="govuk-grid-row">
-            <div class="govuk-grid-column-one-half">
-                {{ form_known_date(form.startDate, 'form.startDate') }}
-            </div>
-            <div class="govuk-grid-column-one-half">
-                {{ form_known_date(form.endDate, 'form.endDate') }}
-            </div>
-            </div>
-        {% endset %}
-
-        {{ form_checkbox_group(form.period, 'form.period', {
-            classes: 'govuk-radios--small',
-            legendClass: 'govuk-visually-hidden',
-            items: [
-            {},
-            {},
-            {},
-            { conditional: customDateRange }
-            ]
-        }) }}
-
-        {{ form_submit(form.update, 'form.update', { buttonClass: 'govuk-!-margin-bottom-0' }) }}
-
-        {{ form_end(form) }}
+                </summary>
+                <div class="govuk-details__text">
+                    {{ form_start(form) }}
+
+                    {% set customDateRange %}
+                        <div class="govuk-grid-row">
+                            <div class="govuk-grid-column-one-half">
+                                {{ form_known_date(form.startDate, 'form.startDate') }}
+                            </div>
+                            <div class="govuk-grid-column-one-half">
+                                {{ form_known_date(form.endDate, 'form.endDate') }}
+                            </div>
+                        </div>
+                    {% endset %}
+
+                    {{ form_checkbox_group(form.period, 'form.period', {
+                        classes: 'govuk-radios--small',
+                        legendClass: 'govuk-visually-hidden',
+                        items: [
+                            {},
+                            {},
+                            {},
+                            { conditional: customDateRange }
+                        ]
+                    }) }}
+
+                    {{ form_submit(form.update, 'form.update', { buttonClass: 'govuk-!-margin-bottom-0' }) }}
+
+                    {{ form_end(form) }}
+                </div>
+            </details>
         </div>
-      </details>
     </div>
-  </div>
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full text--center">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full text--center">
       <span class="govuk-caption-l govuk-!-margin-top-2" id="metric-satisfaction-total-label">
         {{ 'metrics.satisfaction.totalLabel' | trans }}
       </span>
-      <span class="govuk-heading-xl govuk-!-margin-bottom-4 govuk-!-font-size-80" aria-labelledby="metric-satisfaction-total-label">
+            <span class="govuk-heading-xl govuk-!-margin-bottom-4 govuk-!-font-size-80"
+                  aria-labelledby="metric-satisfaction-total-label">
         {{ stats.satisfaction.all | default('-') }}%
       </span>
```

Comment:
Formatting issue here. While this is nothing major the indentations make it easier to see where a HTML element starts and finishes :) seems like this `span` needs a tab over 👍🏻